src/box/sql/select.c

```diff
@@ -4955,6 +4955,14 @@ selectExpander(Walker * pWalker, Select * p)
 			struct space *space = sql_lookup_space(pParse, pFrom);
 			if (space == NULL)
 				return WRC_Abort;
+			if ((selFlags & SF_View) &&
+			    space_is_meta_temporary(space)) {
+				diag_set(ClientError, ER_UNSUPPORTED,
+					 "CREATE VIEW",
+					 "meta-temporary spaces");
```

Comment:
Why do you need to handle it here instead of `alter.cc`? If you handle it just here, then can I still violate it via manual insertion into `box.space._space` or what? If I can't, then what is the point of having this check here?

Reply:
fixed